debian/tests/common-functions

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
```

Comment:
Probably a good idea to run "shellcheck" over this script (and ideally make it part of the unit tests to run shellcheck on this and all other shell scripts).

Reply:
It is already run by check-all-the-things, thus adding it per package feels redundant.

Reply:
But thanks, I used it to check the scripts.

Reply:
Aha, thank you! I was not aware that is already used, nice!